### PR TITLE
Trailing zeroes

### PR DESCRIPTION
**Pull request by NEERAJAP2001**
Solves #111

### CODE REVIEW

**Comment by NEERAJAP2001** [comment]
@rcoder23 This is the new PR, I've created a branch called"Traling_Zeroes" .

File: Numerical/Trailing_Zeroes.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        number //= 5
+        # add to rem
+        rem += number
+    # rem has the answer
+    return rem
+    
+    
+# taking input
+number = int(input('Enter Number for calculating trailing Zeroes : '))
+
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:17]: Remove extra line

File: Numerical/Trailing_Zeroes.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    # rem has the answer
+    return rem
+    
+    
+# taking input
+number = int(input('Enter Number for calculating trailing Zeroes : '))
+    
+# gtting the solution
+solution = trailingZeroes(number)
+
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:20]: Remove extra line

File: Numerical/Trailing_Zeroes.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Function to get trailing zeroes
+def trailingZeroes(number):
+    #initialise to 0
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:3]: Self explanatory No need to explain each and every line

File: Numerical/Trailing_Zeroes.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Function to get trailing zeroes
+def trailingZeroes(number):
+    #initialise to 0
+    rem = 0
+    # Till Number is graeter than zero loop will execute
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:5]: Self explanatory

File: Numerical/Trailing_Zeroes.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Function to get trailing zeroes
+def trailingZeroes(number):
+    #initialise to 0
+    rem = 0
+    # Till Number is graeter than zero loop will execute
+    while number > 0:
+        # keep diving by 5
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:7]: Self explanatory 
Don't write unless specific

File: Numerical/Trailing_Zeroes.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Function to get trailing zeroes
+def trailingZeroes(number):
+    #initialise to 0
+    rem = 0
+    # Till Number is graeter than zero loop will execute
+    while number > 0:
+        # keep diving by 5
+        number //= 5
+        # add to rem
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:9]: Self explanatory

File: Numerical/Trailing_Zeroes.py [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+def trailingZeroes(number):
+    #initialise to 0
+    rem = 0
+    # Till Number is graeter than zero loop will execute
+    while number > 0:
+        # keep diving by 5
+        number //= 5
+        # add to rem
+        rem += number
+    # rem has the answer
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:11]: Self explanatory

File: Numerical/Trailing_Zeroes.py [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    return rem
+    
+    
+# taking input
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:15]: Self explanatory

File: Numerical/Trailing_Zeroes.py [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+number = int(input('Enter Number for calculating trailing Zeroes : '))
+    
+# gtting the solution
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:18]: Self explanatory 
Don't make the code long

File: Numerical/Trailing_Zeroes.py [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+solution = trailingZeroes(number)
+
+# printing to console
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:21]: Self explanatory

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

File: Numerical/Trailing_Zeroes.py [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+solution = trailingZeroes(number)
+print(solution)
+
+# Input: number = 3
+# Output: 0
+# Explanation: 3! = 6, no trailing zero.
+
+# Input: number = 5
+# Output: 1
+# Explanation: 5! = 120
```
Comment (rohanbabbar04) [inline Numerical/Trailing_Zeroes.py:19]: Use docstrings (''') instead of #

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

**Review by rohanbabbar04 — APPROVED** [review]

**Review by rcoder23 — APPROVED** [review]

**Pull request merged**